CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+Possible log types are `[add]`, `[bug]`, `[change]` and `[info]`.
```

Comment:
Ah I see.
However, having a summary of *which* changes were breaking and require changes on the client side may be of interest while reading a change log. BTW: Found an interesting [link](http://keepachangelog.com/).